src/_common.scss

```diff
@@ -14,17 +15,22 @@
 
 .btn-filled {
   background: $news-button-fill;
+  background: var(--colour-button, $news-button-fill);
   color: #000;
+  color: var(--colour-button-text, $news-button-fill);
```

Comment:
This looks like a button might end up with the same colour as its background.

Reply:
Good catch, have updated that.